docs/data/material/migration/migrating-from-deprecated-apis/migrating-from-deprecated-apis.md

```diff
@@ -1093,3 +1093,33 @@ Here's how to migrate:
    },
  },
 ```
+
+## Tooltip
+
+Use the [codemod](https://github.com/mui/material-ui/tree/HEAD/packages/mui-codemod#tooltip-props) below to migrate the code as described in the following sections:
+
+```bash
+npx @mui/codemod@latest deprecations/tooltip-props <path>
```

Comment:
```suggestion
npx @mui/codemod@next deprecations/tooltip-props <path>
```

Reply:
I can see that some docs for component have `latest` and others have `next`. For example `Slider` (example PR for the main issue) has `latest`. I will fix it for my PRs